packages/go/trynoports/app.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+// Model / Init part of Elm architecture for app
```

Comment:
Elm architecture has 3 parts:

1. Model / Init (i.e. state)
2. Update (i.e. update the state based on an event)
3. View (i.e. presentation based on the state)